docs/reference/modules/asset.md

```diff
@@ -29,7 +29,7 @@ By default, `watch` is set to `true`. A truthy value will cause the application
 
 ### `uploadfs`
 
-By default, uploadfs will use [sharp](https://www.npmjs.com/package/sharp) for static asset storage and delivery during the build process and during upload by the `attachment` module. Setting the `image` subproperty to `imagemagick` will force use of this processor if it is installed for the build process. Finally, a fully custom processor can be used by passing an object.  The processor choice for asset upload can be set seperately in the `attachment` module options. Full documentation for uploadfs can be found [here](https://www.npmjs.com/package/uploadfs).
+This optional property can be used to configure an `uploadfs` instance that differs from the one configured by the `attachment` module, allowing changes in where media is stored and how it is served. Full documentation for uploadfs can be found [here](https://www.npmjs.com/package/uploadfs).
```

Comment:
Not media, the asset module doesn't store media, it stores static assets from the webpack build process.

Reply:
Apostrophe will copy the CSS, JS and other build artifacts from the asset built task to uploadfs when the `APOS_UPLOADFS_ASSETS` environment variable is set to `1`.  When the environment variable is set, assets from the build process such as CSS and JS files are copied to uploadfs to allow delivery in multiserver configurations. Otherwise the assets built by the asset module are served from the `public/apos-frontend` folder of the application.

When `APOS_UPLOADFS_ASSETS` is set to `1`, it becomes possible to set the `uploadfs` option of this module to pass a custom uploadfs configuration that differs from that normally shared by the attachment module, which handles user-uploaded media.

Reply:
Changed.